response.h

```diff
@@ -24,12 +24,15 @@ class Response {
     NOT_FOUND = 404,
     FOUND = 302
   };
-  
+
   void SetStatus(const ResponseCode response_code);
```

Comment:
Would this be a good time to refactor the request and response method names to follow our lowerCamel case conventions?